catalog/solr/docs/src/main/resources/ManContents.adoc

```diff
@@ -1,24 +1,422 @@
+It is a local instance that is a lightweight solution that works out of the box without any configuration. However, it does not provide a http://wiki.apache.org/solr/SolrAdminGUI[Solr Admin GUI] or a "REST-like HTTP/XML and JSON API." If that is necessary, see Standalone Solr Server.
+
+==== Installing and Uninstalling
+
+===== Prerequisites
+
+Before the {branding} Solr Catalog Application can be installed:
+
+* the {branding} Kernel must be running,
+* the {branding} Platform Application must be installed, and
+* the {branding} Catalog Application must be installed
+
+===== Installing
+
+. Navigate to the {branding} Solr Catalog application in the Admin console.
+. Under the *Features* tab, uninstall the `catalog-solr-external-provider` feature.
+. Install the `catalog-solr-embedded-provider`
+
+==== Configuring Embedded Solr Server and Solr Catalog Provider
+
+No configuration is necessary for the embedded Solr Server and the Solr Catalog Provider.
+The standard installation described above is sufficient.
+When the `catalog-solr-embedded-provider` feature is installed, it stores the Solr index files to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr` by default.
+A user does not have to specify any parameters and the `catalog-solr-embedded-provider` feature contains all files necessary for Solr to start the server. 
+
+However, this component _can_ be configured to specify the directory to use for data storage. 
+
+The configurable properties for the SCP are accessed from the *Catalog Embedded Solr Catalog Provider* configurations in the Web Console.
+
+[TIP]
+====
+The Embedded (Local) Solr Catalog Provider works on startup without any configuration because a local embedded Solr Server is automatically started and pre-configured.
+====
+
+===== Configurable Properties
+
+[cols="1,1,1,4a,1,1" options="header"]
+|===
+
+|Title
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Data Directory File Path
+|`dataDirectoryPath`
+|String
+|Specifies the directory to use for data storage. The server must be shutdown for this property to take effect. If a filepath is provided with directories that don't exist, SCP will attempt to create those directories. Out of the box (without configuration), the SCP writes to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr`.
+
+If `dataDirectoryPath` is left blank (empty string), it will default to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr`.
+
+If data directory file path is a relative string, the SCP will write the data files starting at the installation directory. For instance, if the string `scp/solr_data` is provided, the data directory will be at `<DISTRIBUTION_INSTALLATION_DIRECTORY>/scp/solr_data`.
+
+If data directory file path is `/solr_data` in Windows, the Solr Catalog Provider will write the data files starting at the beginning of the drive, e.g., `C:/solr_data`.
```

Comment:
Note: Windows paths display backslashes (\) instead of forward slashes (/), but most distributions understand/translate between them so not sure we want to change all instance of `C:/...` to `C:\...`... but most of the document shows backslashes... most of the solr section shows forward slashes.